docs/pages/project/contributing/contributing-docs.md

```diff
@@ -147,6 +147,35 @@ Meshery documentation is made of these components:
 
   _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artifacts for production._
 
+### Using Docker
+If you've Docker and `make` installed in your system, then you can serve the site locally
+  ```
+  $ make docker-docs
+  ```
+
+This doesn't require the need for installing Jekyll and Ruby in your system
+
+**But, you need to make sure that GNU make is working in your system (might not work in Windows)** 
+#### Note
+While performing the above step, if you're facing errors with a message like below...
+
+`Your ruby version is x.x.x but your Gemfile specified 2.7.x`
+
+This is because Jekyll always considers the exact version of Ruby unlike JavaScript.
+
+So, you need to follow either of the two steps to resolve this problem;
+  - Install the required Ruby version by using `rvm` or by any means given above
+  - Alternatively, if you have Docker installed, then type `make docker-docs` to view the changes
+  - If you're unable to install the required Ruby version, then manually configure the `Gemfile` as below (do only if you have Ruby version <3.0):
```

Comment:
Hey @alphaX86 I think we can specify it as `(not recommended)` step because this step can create some problem. So basically we can suggest out contributor to follow any of first two steps but if they cannot solve the problem with those two steps then as a last option they can hop on to this step. What your view on this @alphaX86

Reply:
Alright, then I'll mark this as `not recommended`